tests/phpunit/tests/option/networkOption.php

```diff
@@ -56,6 +56,30 @@ public function test_delete_network_option_on_only_one_network() {
 		$this->assertSame( $value, get_network_option( $id, $option, false ) );
 	}
 
+	/**
+	 * Tests that calling delete_option() update nooptions when option deleted.
+	 *
+	 * @group ms-required
+	 *
+	 * @ticket 61484
```

Comment:
```suggestion
	 * @ticket 61484
```

With the cache group suggestion, this can run on both single and multisite installs,